pkg/signup/service/signup_service_test.go

```diff
@@ -76,9 +76,8 @@ func (s *TestSignupServiceSuite) TestSignup() {
 
 		val := userSignups.Items[0]
 		require.Equal(s.T(), configuration.Namespace(), val.Namespace)
-		require.Equal(s.T(), userID, val.Name)
-		require.Equal(s.T(), userID, val.Spec.Userid)
```

Comment:
we should verify that the `Spec.UserID` is correctly set as well

Reply:
No problem, done.